react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+function PokemonList() {
+  const listPokemon = pokedex.map(pokemon =>
```

Comment:
Since you are using a functoin component you should get the pokedex data from props. So instead of `pokedex.map(...)` you should add `props` to your parameter list then you would do `props.pokemon.map(...)`